cmd/captplanet/setup.go

```diff
@@ -42,6 +43,13 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
 	if !setupCfg.Overwrite && !valid {
 		return fmt.Errorf("captplanet configuration already exists (%v). Rerun with --overwrite", setupDir)
 	} else if setupCfg.Overwrite && err == nil {
+		opts := []string{"yes", "no"}
+		question := fmt.Sprintf("Are you sure you want to delete the following directory?\n%v", setupDir)
+		i := prompt.Choose(question, opts)
```

Comment:
Cant we choose the Confirm Option of the module here and get rid of the above declaration of Yes/No?
This is also breaking our tests as Travis isn't responding interactive.

Reply:
should be as simple as this:
```
if ok := prompt.Confirm("Are you sure you want to delete the following directory?\n%v", setupDir); !ok {
    return
}
```